.github/workflows/rust-cli-publish.yml

```diff
@@ -38,6 +38,7 @@ jobs:
           Native Zowe CLI client which communicates with a "daemon" version of Zowe CLI.
         draft: false
         prerelease: false
+        makeLatest: false
```

Comment:
Will we have to go in and manually change the latest GH Release after every PR is merged (and published)?

Reply:
I don't think we should have to manually set any releases to latest.
The `native-X` tags that we release for the Rust binary in this workflow should never be "latest" if we want the latest release to point to a release of the Zowe CLI package that is created by Octorelease.